make-options.json

```diff
@@ -52,6 +52,7 @@
         "DotNetCoreInstallerV0",
         "DownloadBuildArtifactsV0",
         "DownloadFileshareArtifactsV0",
+        "DownloadGitHubReleasesV0",
```

Comment:
After checkin-ing this task, update owner information as well https://github.com/Microsoft/azure-pipelines-tasks/blob/master/.github/CODEOWNERS